dashboard/app/bisect_test.go

```diff
@@ -246,7 +246,6 @@ https://goo.gl/tpsmEJ#testing-patches`,
 	c.incomingEmail(msg4.Sender, "#syz upstream")
 	msg4 = c.pollEmailBug()
 	c.expectEQ(msg4.To, []string{
-		"bugs2@syzkaller.com",
```

Comment:
Wait, why does it disappear? It's bad if we forget to include some emails now. Or is it on CC now? If so we need to check CC as well. Tests that check To are specifically crafted to check that we send email to all people we need to send it to and don't send to the people we shouldn't send it to. If a test checks To, it now need to check Cc as well.